### DISCUSSION TITLE
Visual Studio 2022 ToolTip color unreadable

### DISCUSSION THREAD

**Discussion by StephaneSuzy** [community/community#18273] [Copilot Conversations]
Is there a setting that lets me adjust the color for the editor tooltip?  I tried the usual 

Tools - Options - Fonts and Colors
 - Show settings for:  Environment
 - Show settings for : Editor Tooltip

Modifying those settings does not get rid of the black background on black text that I am seeing here:
![image](https://user-images.githubusercontent.com/89419534/173093888-fe5a4998-946f-4168-a382-4f1ecfdbda2b.png)

How can I fix this???

**Comment by sejobasic** [comment]
Commenting because i am looking for the same settings

**Reply by dekenless** [reply]
Same

**Reply by linwood-rancho** [reply]
Same Same

**Reply by ayim** [reply]
@linwood-rancho could you file an issue using VS Feedback so our developer can engage?
![image](https://github.com/community/community/assets/10951869/9da526d2-72e9-4e2d-8b59-9aae43ea2d73)
